example-basic/src/main/java/net/optile/example/basic/SdkThemeBuilder.java

```diff
@@ -31,6 +31,8 @@ public static PaymentTheme createDefaultTheme() {
      */
     public static PaymentTheme createCustomTheme() {
         return PaymentTheme.createBuilder().
+            setPaymentListTheme(R.style.CustomTheme_Toolbar).
+            setChargePaymentTheme(R.style.CustomTheme).
             build();
```

Comment:
Small improvement of readability: starting a new line with `.` shows immediately that this is a continuation:
```suggestion
        return PaymentTheme.createBuilder()
            .setPaymentListTheme(R.style.CustomTheme_Toolbar)
            .setChargePaymentTheme(R.style.CustomTheme)
            .build();
```